docs/source/developers/ROADMAP_0_4.md

```diff
@@ -0,0 +1,90 @@
+# Roadmap
+
+## For 0.4.* series of releases - November 2020
+
+The napari roadmap captures current development priorities within the project and aims to guide for napari core developers, to encourage and inspire contributors, and to provide insights to external developers who are interested in building for the napari ecosystem. For more details on what this document is and is not, see the [about this document section](#about-this-document).
+
+The [mission](MISSION_AND_VALUES.md#our-mission) of napari is to be a foundational multi-dimensional image viewer for Python and provide graphical user interface (GUI) access to image analysis tools from the Scientific Python ecosystem for scientists across domains and levels of coding experience. To work towards this mission, we have set the following three high-level priorities to guide us over the upcoming months:
+
+- Make the **data viewing and annotation** capabilities **bug-free, fast, and delightful to use**.
+
+- Support **functional and interactive** plugins that allow for image analysis.
+
+- Add accessible documentation, tutorials, and demos.
+
+You can read more about how this roadmap builds on and continues the work in our `0.3` series of releases in our [0.3 lookback section](#lookback-at-the-0.3-roadmap). We're continuing to prioritize the robustness and polish of the core viewer to ensure that plugin development happens against a solid foundation, and because we believe that looking at and annotating your data should be a bug free and delighful experience, regardless of how large it is. We want to build on the success of our reader / writer plugins to provide the ability to add functional and interactive plugins and further empower a lot of the great work being built ontop of napari.
+
+## Make the data viewing and annotation capabilities bug-free, fast, and delightful to use
+
+- ...
+
+- **Better support for viewing big datasets**. Currently, napari is fast when viewing on-disk datasets that can be naturally sliced along one axis (e.g. a time series) *and where loading one slice is fast*. However, when the loading is slow, the napari UI itself becomes slow, sometimes to the point of being unusable. We now have experimental support for asyncronous rendering for our images, and are in the process of adding support for an octree to better support multiscale tiled data for both 2D and 3D rendering.
```

Comment:
Both of these are now implemented but as opt-in, only programmatically. I would outline two priorities here:
- add persistent settings and use those to allow anyone to use these features, even in the bundled app
- bring the features out of experimental and into mainline napari

Reply:
can you take this one